### PR TITLE
Support commonmark

### PR DESCRIPTION
**Pull request by raphink**
Standards, standards, standards... and a nicer look on GitHub, too!

This PR might still require a few adjustments here and there.

One issue I see is that I had to remove the `highlight` CSS class condition for highlighting, because CommonMarker doesn't put it. Can you think of another way to do that properly?

### CODE REVIEW

**Comment by binford2k** [comment]
Thanks @raphink. I'll poke at the highlight issue and merge this for the major release next week :)

**Comment by raphink** [comment]
Cool. Let me know if I can help. For now, I'm going to start using my version as I'm writing a new course and want to use standard markdown :smile:

File: lib/showoff_utils.rb [hunk 1]
```diff
@@ -442,6 +442,10 @@ def self.setup(dir_name)
     when 'kramdown'
       Tilt.prefer Tilt::KramdownTemplate, "markdown"
 
+    when 'commonmarker'
+      require 'tilt/commonmarker'
```
Comment (raphink) [inline lib/showoff_utils.rb:446]: This should not be required once https://github.com/rtomayko/tilt/pull/282 is merged.

Comment (raphink) [inline lib/showoff_utils.rb]: This should not be required once https://github.com/rtomayko/tilt/pull/282 is merged.

Comment (raphink) [inline lib/showoff_utils.rb]: This should not be required once https://github.com/rtomayko/tilt/pull/282 is merged.

**Comment by binford2k** [comment]
As soon as tilt > 2.0.2 is released, we can get this merged. https://github.com/rtomayko/tilt/blob/master/CHANGELOG.md

**Comment by raphink** [comment]
Let's hope that will happen soon then :smile:

**Pull request merged**